go-controller/pkg/util/util.go

```diff
@@ -624,3 +626,8 @@ func GetServiceEndpointSlices(namespace, svcName, network string, endpointSliceL
 	}
 	return GetEndpointSlicesBySelector(namespace, selector, endpointSliceLister)
 }
+
+// IsUDNEnabledService checks whether the provided namespaced name key is a UDN enabled service specified in config.Default.UDNAllowedDefaultServices
+func IsUDNEnabledService(key string) bool {
+	return strings.Contains(config.Default.UDNAllowedDefaultServices.String(), key)
```

Comment:
why is this contains and not equal? configured allowed udn service test/abc could accidentally match service test/ab?

Reply:
:facepalm: 
Fixed.